### ISSUE TITLE
Will Pytorch3D support the ray-tracing method?

### ISSUE THREAD

**Issue by ShengrenNiu** [facebookresearch/pytorch3d#340]
It seems that pytorch3D only support the rasterized renderer. Will it support the  ray-tracing method in the future version?

**Comment by aluo-x** [comment]
So technically [mitsuba2](https://github.com/mitsuba-renderer/mitsuba2) natively supports gradients for pytorch tensors. However there are many functions lacking (batched rendering, multi-gpu, [differentiation w.r.t. vertices](https://github.com/mitsuba-renderer/mitsuba2/pull/157) etc.) that make it not realistic to use for training, but these are supposedly on their roadmap. The documentation is also quite sparse, so your mileage may vary.
